### PR TITLE
Taj Hand Pattern Fixes

### PR DESCRIPTION
**Pull request by WickedCybs**
I was going through the Taj patterns and noticed that for the vast majority, the hand patterns weren't shaded right and in one case, actually gave taj an extra hand, so whoops.

Did some quick and easy fixes, will probably have to take a deeper look later at the rest, these are pretty old patterns after all so I'm not surprised they needed touching up and more fixes.

An example of one of the main changes

This is the hands at default, no markers applied.
image

This is a pattern where, while it doesn't look too bad its not really consistent with the sprite itself. 
image

Basically, just making it more natural among other things. A lot of the time, aspects like the outline were removed by these patterns. That's fixed. Should be overall more consistent.

### CODE REVIEW

**Comment by WickedCybs** [comment]
!review

**Comment by WickedCybs** [comment]
!fix

**Review by Geevies — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]
